GOVERNANCE-subprojects.md

```diff
@@ -0,0 +1,172 @@
+# [TODO:PROJECTNAME] Project Governance
+
+This is an example governance document for projects that consist of a group
+of smaller, semi-independent projects.  See [GOVERNANCE.md](/GOVERNANCE.md)
+for more information.  Thanks to the Konveyor Project for portions of the text
+below.
+
+This is a template document for CNCF projects that requires editing
+before it is ready to use. Read the markdown comments, `<!-- COMMENT -->`, for
+additional guidance. The raw markdown uses `TODO` to identify areas that
+require customization.  Replace [TODO: PROJECTNAME] with the name of your project.
+
+This Template assumes that you are using the Contributor Ladder template as well.
+
+<!-- template begins here-->
+
+The [TODO:PROJECTNAME] umbrella project is composed as a federation of individual projects,
+some independent and some interdependent, each of which focuses on some aspect
+of [TODO:Goals of umbrella project].  Our governance reflects this federated structure.
+
+
+
+## Individual Project Governance
+
+<!-- the below assumes that you want to mandate a simple, maintainer-driven
+governance structure for each subproject, which we recommend.-->
+
+All active Maintainers of each subproject, as defined in the Contributor Ladder, are
+members of that project's Maintainer Committee, which governs that project.  The
+subproject's Maintainer Committee is responsible for the following project governance
+activities:
+
+* Ensuring that the subproject creates and publishes regular releases;
+* Holding regular, subproject-wide discussions on issues and planning;
+* Monthly review of project contributors for advancement on the Contributor Ladder;
+* Making final decisions on subproject changes that involve controversial trade-offs;
+* Responding to security compromise reports;
+* Supporting the Code of Conduct within their subproject and referring violations
+  to the Code of Conduct Committee.
+
+Additionally, the project's Maintainer Committee will select, by majority vote, one
```

Comment:
Additionally, each subproject's Maintainer Committee ... 

I like replacing "the" with "each" in this case to make it clear that every subproject does this independently.